tests/infrastructure/pom.xml

```diff
@@ -58,6 +58,7 @@
         <dependency>
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-exec</artifactId>
+            <version>1.3</version>
```

Comment:
Would be better to update it in the bom

Reply:
i will actually remove it from the bom, since this is used only in these tests